.github/workflows/ci.yml

```diff
@@ -55,7 +55,10 @@ jobs:
       - name: Check that workflows are up to date
         run: sbt ++${{ matrix.scala }} githubWorkflowCheck
 
-      - run: sbt ++${{ matrix.scala }} fmtCheck test mimaReportBinaryIssues
+      - run: |
+          sbt ++${{ matrix.scala }} fmtCheck \
```

Comment:
why do we need this change? Also, it is interesting there is no quote on the first item.